### PR TITLE
[WEB-30][onboarding] skip for organizational accounts

### PR DESCRIPTION
**Pull request by AlexTugarev**
## Description
Attempt to disable the onboarding flow for organizational accounts, i.e. created using OIDC SSO of any org. 

## Related Issue(s)

Fixes WEB-30

## How to test
1. Setup OIDC SSO on a new Org
2. Use the Login button (...) next to the OIDC config entry
3. See that there is no Onboarding flow happening after signing in

## Release Notes

```release-note
NONE
```

## Build Options:

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] with-dedicated-emulation
- [ ] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

**Comment by geropl** [comment]
(edited the title to test the linear integration :point_up: )

File: components/gitpod-protocol/src/protocol.ts [hunk 1]
```diff
@@ -120,6 +120,9 @@ export namespace User {
     }
 
     export function isOnboardingUser(user: User) {
+        if (!!user.organizationId) {
```
Comment (geropl) [inline components/gitpod-protocol/src/protocol.ts:123]: :bulb: Would be great to resort to a method `User`, e.g. `User.isOrganizationalUser` or so. Just to highlight the dependency.

But this works, to unblocking nonetheless!

**Review by geropl — APPROVED** [review]
:heavy_check_mark:

**Comment by AlexTugarev** [comment]
Still doesn't work. Also, `Fixes Y` [should work as well.](https://www.notion.so/gitpod/Linear-for-PDE-Planning-and-Execution-d50258693e5e4adc81ad05a4f9a6325a?pvs=4#bca809aa4617462880b416179e0e5fd7)

**Pull request merged**